test/WebJobs.Script.Tests.Analyzers/WebJobs.Script.Tests.Analyzers.csproj

```diff
@@ -4,10 +4,16 @@
     <TargetFramework>net5.0</TargetFramework>
 
     <IsPackable>false</IsPackable>
+
+	<SignAssembly>true</SignAssembly>
+
+	<AssemblyOriginatorKeyFile>key.snk</AssemblyOriginatorKeyFile>
```

Comment:
nit: line this up with other attributes

Reply:
How weird; it was properly indented when I committed it. Fixing.